locales/de/de.json

```diff
@@ -280,7 +280,7 @@
     "Leave": "Verlassen",
     "Leave Team": "Team verlassen",
     "Lebanon": "Libanon",
-    "Lens": "Lens",
+    "Lens": "Lupe",
```

Comment:
Lens is a proper name (Eigenname) of Laravel Nova. Please do not translate.

Reply:
Ok, then I agree its better not to change it